tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -381,19 +381,19 @@
 
         /**
          * The set of all dashboards. Each object within this array is a
-         * DashboardInformation object.
+         * dashboardData object.
          *
          * @type {!Array<Object>}
          */
-        _dashboardInformationObjects: {
+        _dashboardDataObjects: {
```

Comment:
The other half of my earlier comment (perhaps just implied) was that the `Objects` is totally redundant here. `_dashboardData` is easier to work with, and the "data" bit clarifies that it's not just the dashboard names (a goal that you noted in your original description, and with which I agree).

Reply:
True. Objects seems redundant. We now use `Data` to refer to a list of `Datum`s. Thus, renamed `PLUGIN_TO_DASHBOARD_DATA` to `PLUGIN_TO_DASHBOARD_DATUM`

Reply:
Cool. I was considering that; either's fine with me.